### DISCUSSION TITLE
Reusing AWS credentials for route53 DNS challenge

### DISCUSSION THREAD

**Discussion by TheRealBenForce** [NginxProxyManager/nginx-proxy-manager#3412] [Q&A]
I feel like this should be easier like an OOTB feature but I haven't found a way. How do you all handle reusing AWS credentials when you later go on to add a second service on a second subdomain. I'd prefer not to use a wildcard cert so each time I add a new service on a new subdomain I need to copy and paste the same credentials into npm. Is there an elegant way to handle this and reference existing credentials? I currently just have my access/secret keys in my docker .env file. Thanks!

**Comment by platinumnh** [comment]
yup totally agree, should be saved when used once

**Comment by brwyatt** [comment]
While Proxmox's setup is a bit more complicated because of clustering, something like how they have it where you configure the ACME plugin or create some profile, then select that profile rather than the plugin directly (and thus needing to supply the config directly). That would drastically reduce the complexity here (and having to duplicated the creds in multiple places).
